components/script/dom/htmlimageelement.rs

```diff
@@ -141,11 +196,31 @@ impl HTMLImageElement {
                     let runnable = wrapper.wrap_runnable(runnable);
                     let _ = script_chan.send(CommonScriptMsg::RunnableMsg(
                         UpdateReplacedElement, runnable));
-                });
+                });*/
```

Comment:
It looks like we still need to use ImageResponseHandlerRunnable, since it's responsible for things like the image's `load` event. We can move the call to `request_image_and_metadata` into `response_complete` after calling `store_complete_image_bytes`.